steel/README.md

```diff
@@ -62,12 +65,15 @@ Here is a snippet to the [relevant code](../examples/erc20/host/src/main.rs) on
 // Create a view call environment from an RPC endpoint and a block number. If no block number is
 // provided, the latest block is used. The `with_chain_spec` method is used to specify the
 // chain configuration.
-let mut env = EthViewCallEnv::from_rpc(&RPC_URL, None)?
-    .with_chain_spec(&ETH_SEPOLIA_CHAIN_SPEC);
+let mut env =
+    EthViewCallEnv::from_rpc(&args.rpc_url, None)?.with_chain_spec(&ETH_SEPOLIA_CHAIN_SPEC);
+let number = env.header().number();
+let commitment = env.block_commitment();
 
 // Preflight the view call to construct the input that is required to execute the function in
 // the guest. It also returns the result of the call.
-let returns = env.preflight(ViewCall::new(CALL, CONTRACT).from(CALLER))?;
+let mut contract = Contract::preflight(CONTRACT, &mut env);
+let returns = contract.call_builder(&CALL).from(CALLER).call()?;
 let input = env.into_zkvm_input()?;
```

Comment:
I think we should rename that. It seems a bit arbitrary to have `zkvm` in the name especially since there might also be other usecases of using steel. I think `into_input` is just fine.

Reply:
sure! That seems reasonable

Reply:
Didn't add a deprecation for the old naming since that was never released, but happy to if we'd like it (light weight)